src/sscce-sequelize-6.ts

```diff
@@ -4,7 +4,9 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 
 // if your issue is dialect specific, remove the dialects you don't need to test on.
-export const testingOnDialects = new Set(['mssql', 'sqlite', 'mysql', 'mariadb', 'postgres', 'postgres-native']);
+// This problem is specific to mysql, but ci  mysql verison to low and use group need config setting so can't run test:mysql
```

Comment:
It's using the latest MySQL 8.0 version here;
https://github.com/sequelize/sequelize-sscce/blob/a8ce25ff1f8edb61ade5a9ae3749843e41a20a50/setup/mysql/8.0/docker-compose.yml#L4

So what version of MySQL gives the wrong result?